File: test/fixtures/output/java/restclient/jsonObj-multiline.java
RestClient restClient = RestClient.create();

ResponseEntity<String> response = restClient
  .method(HttpMethod.POST)
  .uri("http://mockbin.com/har")
  .contentType(MediaType.APPLICATION_JSON)
  .body("{\n  \"foo\": \"bar\"\n}")
  .retrieve()
  .toEntity(String.class);

File: test/fixtures/output/csharp/restsharp/jsonObj-multiline.cs
var client = new RestClient("http://mockbin.com");
var request = new RestRequest("/har", Method.Post);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{\n  \"foo\": \"bar\"\n}", ParameterType.RequestBody);
var response = client.Execute(request);